components/tikv_util/src/security.rs

```diff
@@ -144,6 +186,42 @@ impl SecurityManager {
     }
 }
 
+struct Fetcher {
+    certs: Arc<RwLock<Certs>>,
+    last_modified: Arc<RwLock<SystemTime>>,
```

Comment:
I think you can use `Mutex`.

Reply:
`RwLock` here is suitable for multiple reads and fewer writes. I think the read should be shared so that multiple client connection read can not be blocked.

Reply:
The performance of `RwLock` is worse than that of `Mutex`, and the time consuming of read here is not large, so use back `Mutex`.